exporter.go

```diff
@@ -16,13 +16,14 @@ package main
 
 import (
 	"database/sql"
+	"fmt"
 	"math"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/go-kit/log/level"
```

Comment:
Why logger was changed?

Reply:
the new version of the client_go library doesn't have a logger anymore, so I changed it to the one used by other exporters.